runtime-sdk/src/modules/rofl/app/client.rs

```diff
@@ -195,6 +195,11 @@ where
                     propagate_failures: false,
                 })
                 .await?;
+
+            // The estimate may be off due to current limitations in confidential gas estimation.
+            // Inflate the estimated gas by 20%.
+            let gas = gas.saturating_mul(120).saturating_div(100);
```

Comment:
```suggestion
            let gas = gas.saturating_add(gas.saturating_mul(20).saturating_div(100));
```
In theory, this could result in a lower gas.

Reply:
Significantly? The 20% is a rough estimate anyway so it shouldn't matter if it is not exact.

Reply:
Significantly yes, as in theory this value could reduce for a factor of 100, e.g., if `gas = maxu64`, then new gas is `gas = maxu64/100`. In practice probably not possible.